app/design/frontend/Magento/storefront/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -0,0 +1,156 @@
+//
+//  Common
+//  _____________________________________________
+
+& when (@media-common = true) {
+    .page-wrapper{
+        .page-main{
+            max-width: 1920px;
+            padding-left:0;
+            padding-right:0;
+        }
+    }
+
+    .baner {
+        width: 100vw;
+        height: 75vh;
+        max-height: 928px;
+        background-color: #F3F3F3;
+        align-items: center;
+        display: flex;
+    }
+    .container {
+        margin: 0 auto;
+        width: 60%;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        max-width:700px;
+        img {
+            width: 40%;
+            height: max-content;
+            max-width:300px;
+        }
+    }
+    .container_text {
+        width: inherit;
+        display: flex;
+        flex-direction: column;
+        * {
+            font-family: 'Poppins', sans-serif;
+            &:nth-child(odd) {
+                text-transform: uppercase;
+            }
+            &:first-child {
+                font-size: 27px;
+                letter-spacing: 2px;
+                font-weight: lighter;
+                >span {
+                    font-weight: 600;
+                }
+            }
+            &:nth-child(2) {
+                font-size: 10px;
+                color: #a3a3a3;
+                letter-spacing: 1.1px;
+                margin-bottom: 30px;
+            }
+            &:nth-child(3) {
+                text-decoration: none;
+                cursor: pointer;
+                font-size: 11px;
+                letter-spacing: 1px;
+                color: #000;
+                border-bottom: #000 solid 1px;
+                width: max-content;
+                padding-bottom: 3px;
+                &:hover {
+                    text-decoration: none;
+                    color: #000;
+                }
+            }
+        }
+    }
+}
+
+
+
+//  Desktop
+//  _____________________________________________
+
+.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__l) {
+    .block{
+        .block-static-block{
+            .baner{
+                .container{
+                    width:65%;
+                }
+            }
+        }
+    }
+}
+
+.media-width(@extremum, @break) when (@extremum = 'min') and (@break = 970px) {
```

Comment:
create custom breakpoint for @break = 970px, 870px and 540px
https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/responsive-web-design/rwd_css.html